lib/archethic/p2p.ex

```diff
@@ -102,6 +102,26 @@ defmodule Archethic.P2P do
     end
   end
 
+  @doc """
+  Called by the telemetry poller
+  """
+  def nodes_connected_count() do
+    nodes_connected =
+      Archethic.P2P.authorized_and_available_nodes()
```

Comment:
Wondering if it should be `list_nodes` or `authorized_and_available_nodes`.
Also you can remove `Archethic.P2P`
```suggestion
      authorized_and_available_nodes()
```